superset/viz.py

```diff
@@ -2107,6 +2108,10 @@ def tupleify(s):
             df[key] = list(zip(latlong.apply(lambda x: x[0]),
                                latlong.apply(lambda x: x[1])))
             del df[spatial.get('geohashCol')]
+
+        if df[key] is None:
+            raise NullValueException('Some rows in this query contain NULL values!')
```

Comment:
Msg could be a bit more precise and use i18n wrapper: `_(Encountered invalid NULL spatial entry, please consider filtering those out)`

Reply:
@vylc wording ok?

Reply:
Is there a python lib for ` i18n`? i thought is was only available for JS, i was searching the code for some example and couldn't find any.

Reply:
in this module it's available from the line `from flask_babel import lazy_gettext as _`

Reply:
the setup is kind of sweet where we use the same framework for both the backend and frontend.